packages/xl-multi-column/package.json

```diff
@@ -65,7 +73,9 @@
     "typescript": "^5.3.3",
     "vite": "^5.3.4",
     "vite-plugin-eslint": "^1.8.1",
-    "vitest": "^2.0.3"
+    "vitest": "^2.0.3",
+    "react": "^19",
```

Comment:
should this be ^18?

Reply:
I put 19 to test against that, but 19 is stable now so it shouldn't matter which